commands/command_filter_process.go

```diff
@@ -206,9 +206,9 @@ func filterCommand(cmd *cobra.Command, args []string) {
 	}
 
 	if len(malformed) > 0 {
-		fmt.Fprintf(os.Stderr, tr.Tr.GetN(
-			"Encountered %d file that should have been pointers, but wasn't:\n",
-			"Encountered %d files that should have been pointers, but weren't:\n",
+		fmt.Fprintln(os.Stderr, tr.Tr.GetN(
+			"Encountered %d file that should have been pointers, but wasn't:",
```

Comment:
Should we also have changed this to be "should have been a pointer"?

Reply:
Yup, good eye, thank you!  Fixed.